jQuery/ajax.js

```diff
@@ -21,6 +33,50 @@ function getUniqueValuesForField(field, citizens) {
 
 function populateSelectWithOptions(select, options) {
   options.forEach(function(option) {
-    $(select).append($("<option>", { value: option, html: option }));
+    $(select).append($('<option>', { value: option, html: option }));
+  });
+}
+
+function gatherParams() {
+  var params = [];
+
+  var characterValue = $('#character-select').val();
+  params.push({field: 'character', value: characterValue});
+
+  var hometownValue = $('#hometown-select').val();
+  params.push({field: 'hometown', value: hometownValue});
+
+  var sizeValue = $('#size-select').val();
+  params.push({field: 'size', value: sizeValue});
+
+  return params;
+}
+
+function filter(params, citizens) {
```

Comment:
It's looping through all citizens and returning a list of those that meet the parameters that were gathered by the ```gatherParams()``` function. It's written in such a  way that if Briana decided to add an extra info field for each citizen (which could definitely happen at some point) the filtering function wouldn't have to change, I'd just have to add a new form element and tell ```gatherParams()``` how to get the new field.

I'm actually going to do a bit more work on this now, as I'd like to add a freeform text input field so she can search on the names. I'll have to change some things here for that